rest_framework/pagination.py

```diff
@@ -534,7 +540,7 @@ def get_next_link(self):
         if not self.has_next:
             return None
 
-        if self.cursor and self.cursor.reverse and self.cursor.offset != 0:
+        if self.cursor and self.cursor.reverse and self.cursor.offset != 0 and self.page:
```

Comment:
Absolutely. But it is impossible to write test for custom cutoff, because of error in this place.

So, you'll ask me to split this PR to two, tel me what both are unacceptable and then accept both again?